apps/src/pythonlab/layout/HorizontalLayout.tsx

```diff
@@ -104,6 +114,25 @@ const HorizontalLayout: React.FunctionComponent<LayoutProps> = ({
             setOutputHeight={setRightBottomPanelSize}
           />
         </div>
+        {aiTutor2Available && (
```

Comment:
Is it intended that ai tutor only shows up in horizontal layout? I think it will disappear when you switch layouts?

Reply:
Good question.  This was intentional, since I had assumed we'd be okay with only showing it in the horizontal layout for initial testing.  cc @samantha-code for awareness.